### ISSUE TITLE
【BUG】Plume.js 2.1 平台軟件 ，windows下無法生成某些拼音文件。

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lotem/zime#23]
```
On windows, certain device names are reserved. For instance, you can't create a 
file named "nul", "nul.txt" (or nul.anything in fact) The reserved names are:

CON, PRN, AUX, NUL, COM1, COM2, COM3, COM4, COM5, COM6, COM7, COM8, COM9, LPT1, 
LPT2, LPT3, LPT4, LPT5, LPT6, LPT7, LPT8, and LPT9


解決辦法：

修改make-json.py文件第 379 行：
        files[index] = 'idx_' + index.encode('utf-7')
这样就行了
```

Original issue reported on code.google.com by `beno...@126.com` on 15 Aug 2010 at 12:57

**Comment by GoogleCodeExporter** [comment]
```
感谢！
http://stackoverflow.com/questions/295135/turn-a-string-into-a-valid-filename-in
-python
```

Original comment by `chen....@gmail.com` on 15 Aug 2010 at 1:07
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed; a new release (v2.2) is due.
```

Original comment by `chen....@gmail.com` on 15 Aug 2010 at 2:45
- Changed state: **Fixed**
